babi/prompt.py

```diff
@@ -14,12 +16,20 @@
 
 
 class Prompt:
-    def __init__(self, screen: Screen, prompt: str, lst: list[str]) -> None:
+    def __init__(
+        self,
+        screen: Screen,
+        prompt: str, lst: list[str],
+        file_glob: bool = False,
```

Comment:
this is a boolean trap -- it should be named-only if anything

Reply:
Sorry, not sure I understand what exactly you'd like me to do here.

Something like `lst: list[str], *, file_glob: bool[ = False]`?

Do you want me to drop the default value as well?

Reply:
hi, I hope my comment doesn't come out of place -- anthony made a [video on boolean traps](https://www.youtube.com/watch?v=CnRkXO_a5mI)